src/Microsoft.VisualBasic/tests/ConversionsTests.cs

```diff
@@ -5,83 +5,2952 @@
+        }
+
+        public static IEnumerable<object[]> ToInteger_InvalidObject_TestData()
+        {
+            yield return new object[] { char.MinValue };
+            yield return new object[] { (char)1 };
+            yield return new object[] { char.MaxValue };
+            yield return new object[] { new DateTime(10) };
+            yield return new object[] { new object() };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToInteger_InvalidObject_TestData))]
+        [MemberData(nameof(InvalidString_TestData))]
+        [MemberData(nameof(InvalidBool_TestData))]
+        public void ToInteger_InvalidObject_ThrowsInvalidCastException(object value)
+        {
+            Assert.Throws<InvalidCastException>(() => Conversions.ToInteger(value));
+        }
+
+        public static IEnumerable<object[]> ToInteger_OverflowObject_TestData()
+        {
+            yield return new object[] { uint.MaxValue };
+            yield return new object[] { (UIntEnum)uint.MaxValue };
+            yield return new object[] { ulong.MaxValue };
+            yield return new object[] { (ULongEnum)ulong.MaxValue };
+            yield return new object[] { long.MinValue };
+            yield return new object[] { long.MaxValue };
+            yield return new object[] { (LongEnum)long.MinValue };
+            yield return new object[] { (LongEnum)long.MaxValue };
+            yield return new object[] { float.MinValue };
+            yield return new object[] { float.MaxValue };
+            yield return new object[] { float.PositiveInfinity };
+            yield return new object[] { float.NegativeInfinity };
+            yield return new object[] { float.NaN };
+            yield return new object[] { double.MinValue };
+            yield return new object[] { double.MaxValue };
+            yield return new object[] { double.PositiveInfinity };
+            yield return new object[] { double.NegativeInfinity };
+            yield return new object[] { double.NaN };
+            yield return new object[] { decimal.MinValue };
+            yield return new object[] { decimal.MaxValue };
+            yield return new object[] { "18446744073709551616" };
+            yield return new object[] { "1844674407370955161618446744073709551616" };
+            yield return new object[] { double.PositiveInfinity.ToString() };
+            yield return new object[] { double.NegativeInfinity.ToString() };
+            yield return new object[] { double.NaN.ToString() };
+        }
+
+        [Theory]
+        [MemberData(nameof(ToInteger_OverflowObject_TestData))]
+        public void ToInteger_OverflowObject_ThrowsOverflowException(object value)
+        {
+            Assert.Throws<OverflowException>(() => Conversions.ToInteger(value));
+        }
+
+        public static IEnumerable<object[]> ToULong_String_TestData()
+        {
+            yield return new object[] { null, ulong.MinValue };
+            yield 
```

Comment:
use inlineData

Reply:
Not anymore, as we share this data with other methods